action.yml

```diff
@@ -115,6 +115,12 @@ runs:
           stack_arguments+=( --pedantic )
         fi
 
+        if [[ "${{ inputs.stack-yaml }}" == 'stack-nightly.yaml' ]]; then
+          if [[ "${stack_arguments[*]}" != *'--resolver'* ]]; then
```

Comment:
:exploding_head: does this do a `not in`?

Reply:
With tons of caveats, but yeah lol